### PR TITLE
Added `getDirtyWithTranslations` method

### PR DESCRIPTION
**Pull request by krecebli**
#373 
I could only add another method `getDirtyWithTranslations`, but I had to make `$this->saveTranslations()` before the `parent::save($options)`. So, it works only on update.

### CODE REVIEW

**Comment by dimsav** [comment]
Please add tests, otherwise I cannot review your changes.

File: src/Translatable/Translatable.php [hunk 1]
```diff
@@ -464,10 +457,50 @@ public function replicateWithTranslations(array $except = null)
...
+        unset($dirty[$this->getLocaleKey()]);
+
+        if ($notEmpty = ! empty($dirty)) {
+            $original = [];
+
+            foreach ($dirty as $key => $value) {
+                $original[$key] = $translation->getOriginal($key);
+            }
+
+            config([
```
Comment (Gummibeer) [inline src/Translatable/Translatable.php]: Please don't feel offended - but I have to say it:
> What the heck!?

Ok, now more objective: why do you put these values into the config? The app config isn't a place for "poor-man"-cache.

File: src/Translatable/Translatable.php [hunk 2]
```diff
@@ -464,10 +457,50 @@ public function replicateWithTranslations(array $except = null)
...
+    }
+
+    /**
+     * Get the attributes that have been changed since last sync.
+     *
+     * @return array
+     */
+    public function getDirtyWithTranslations()
+    {
+        if ($translations = config(static::class.'.'.$this->getKey())) {
```
Comment (Gummibeer) [inline src/Translatable/Translatable.php]: I haven't tested it but this method is public and relies on the config setting in a protected one but doesn't call this method itself.

In my opinion it should be the other way. That the is method is based on the get one.

**Comment by Gummibeer** [comment]
closed due to inactivity

**Pull request closed without merging**